tests/tqueue_general_tests.cpp

```diff
@@ -0,0 +1,101 @@
+#include <gtest/gtest.h>
+#include "tqueue.h"
```

Comment:
I would say choose one way and stick with it

Reply:
Done, decided for "